config/jobs/kubernetes/node-problem-detector/node-problem-detector-presubmits.yaml

```diff
@@ -63,11 +61,10 @@ presubmits:
         args:
         - bash
         - -c
-        - >
-          ./test/build.sh pr $(PULL_REFS) &&
+        - ./test/build.sh pr $(PULL_REFS) &&
           source pr.env &&
           /workspace/scenarios/kubernetes_e2e.py
-          --extract=ci-latest
```

Comment:
this should be ci/latest

Reply:
also there are more `--extract=ci-latest` references in other jobs you might also want to fix

Reply:
Even for presubmit jobs?

Done

Reply:
I assume you only want to build npd but not kubernetes? I forgot what's the intended behavior :-p

Reply:
That's right. Makes sense to use `ci/latest` here.